### DISCUSSION TITLE
Cannot post error in iOS app

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#7] [Q&A] [answered]
The iOS app displayed "Cannot post" followed by "game" or "play" (I forgot) momentarily and then quit.  It did not state why or give the user a chance to retry.

**Comment by azadbolour** [comment, marked as answer]
This error occurs if the internet connection is temporarily lost. 

At this time, the app does not have a way of reconnecting, and simply gives up.  

A fix that will attempt to reconnect in such situations is currently being tested, and is intended to be deployed in April/May 2023 timeframe. 

Please stay tuned.

Note, however, that if the network connection is lost for many seconds, the app will give up trying to reconnect, and will have to exit. 

Of course, the user must be properly informed that repeated attempts to re-establish the network connection have failed, and because the app cannot function without a connection to the EagerWords server, it has to exit.

Azad

**Reply by danconde** [reply]
okay.  I was not on a VPN, so it must have been a temporary network error.

**Comment by azadbolour** [comment]
The iOS beta version 1.2(1) includes the above fix. Once tested, it is slated for release in May 2023.

**Comment by danconde** [comment]
The beta version seems to behave better, from either sleeping and waking, as well as disconnecting from Wi-Fi, using mobile network, and then reconnecting to Wi-Fi, and resuming game.
